lib/templates/core-cf.json

```diff
@@ -3,10 +3,7 @@
   "Description": "The AWS CloudFormation template for this Serverless application's resources outside of Lambdas and Api Gateway",
   "Resources": {
     "coreBucket": {
```

Comment:
Can you rename this from coreBucket to "ServerlessDeploymentBucket" or something similar? coreBucket doesn't really say a lot